### ISSUE TITLE
Reg Documentation , or how to setup with LimeSDR

### ISSUE THREAD

**Issue by abanerjee123** [nqrduck/nqrduck#67]
Hi
           Thanks for the nice project. Actually we are trying to use nqrduck for make some nqr experiments at low frequency ~800 KHz. Is there any documentation? I am not able to compile documentation because pip is not able to fine quackseq . It will be better if you have a flow chart to setup nqrduck with limesdr. Do we need to upload any firmware or any driver to communicate with limeSDR? How one use the limedriver? any help will be appreciated .

Thanks
Abhishek

**Comment by abanerjee123** [comment]
solved

**Comment by jupfi** [comment]
Hi, yes sorry I should add quackseq to PyPi.
The LimeSDR is quite terrible below 30MHz (the matching network in the receive side has terrible performance at lower frequencies): https://wiki.myriadrf.org/LimeSDR_HF_Performance
With the modifications described in the article you can get it lower but I doubt 800kHz would be possible (for the LimeSDR USB). 

Yeah I should add a quickstart guide - you are correct. You only need to upload the Gateware to the LimeSDR if you need the TTL signals (e.g. for a TX gate or TX/RX switches).  Otherwise the LimeSDR in combination with quackseq and nqrduck should work out of the box. This can be done using the LimeSuite and the FPGA bitstream by Andrin Doll: https://sourceforge.net/projects/limr/

But I strongly suspect that using something like the RedPitaya is a better choice at lower frequencies. I wanted to add the RedPitaya to the nqrduck for a long time but haven't gotten around to it in the last months. I've done pulsed NMR spectroscopy using the RedPitaya in the past and it would be quite straight forward. I'll get back to you in case I get it implemented soon :).

**Comment by abanerjee123** [comment]
Thanks for the reply.
I compiled the documents without quackseq .  Now at least I have some documents. I have tested LimeSDR mini 2.4 with NQRduck for higher frequency with loopback measurement. I have some signal but not sure how it should look.

Yes I have seen the gateware from Doll. I will try that.

Yes red pitaya 125-14 looks good and has more capabilities i think. Testing with Pavel firmware. Yes it will be great if you kindly add red pitaya in nqrduck. Thanks.
